geoextent/lib/extent.py

```diff
@@ -99,7 +99,8 @@ def run(self):
             elif self.task == "tbox":
                 try:
                     if tbox:
-                        metadata["tbox"] = usedModule.getTemporalExtent(filePath)
+                        extract_tbox = usedModule.getTemporalExtent(filePath)
```

Comment:
Please check if you can improve the `getTemporalExtent()` function. Does it reliably work without flexible parsing methods? What about parsing to a proper data object already within the `get..` function and then in the next line only turning that date object into the desired string representation?

Reply:
I implemented this approach in the next commit !